extensions/wikia/CommunityPage/CommunityPage.i18n.php

```diff
@@ -99,29 +99,7 @@
 	'communitypage-todo-module-heading' => 'Community to-do list',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "communitypage-todo-module-heading" 13 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)